### PR TITLE
Enhance profile image

### PR DESCRIPTION
**Pull request by poacosta**
## What does this PR do?

* Take a reference image and generate a similar variation. While it's not directly editing, it's equally image manipulation using AI without requiring the user to generate masks to guide the image editing.

Sample:

imagen

Related doc:
*  [Image generation with DALL-E](https://platform.openai.com/docs/guides/images/image-generation)

### CODE REVIEW

File: app/api/enhance/route.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import {NextRequest, NextResponse} from 'next/server'
+import {OpenAI} from 'openai'
+
+const openai = new OpenAI({
```
Comment (marcusrbrown) [inline app/api/enhance/route.ts]: This line should be moved to the top of the route handler function. That should resolve the build issue. 

Also, we don't need to pass in the API key explicitly as long as you have the variable in your `.env.local`.

**Review by marcusrbrown — CHANGES_REQUESTED** [review]

File: app/api/enhance/route.ts (hunk 1 above)
Comment (poacosta) [reply app/api/enhance/route.ts]: On it!

Comment (poacosta) [reply app/api/enhance/route.ts]: Done!

**Review by marcusrbrown — APPROVED** [review]
Awesome, thanks!

**Pull request merged**